docs/src/code/docs-style-guide.adoc

```diff
@@ -1,185 +1,256 @@
 * https://gitlab.com/dashboard/merge_requests?scope=all&state=merged&label_name[]=tw-style&not[label_name][]=docs%3A%3Afix[Recent updates to this guide]
+////
+// Preamble End
+
+[[docs:style:ssot]]
+== Single Source Of Truth (SSOT)
 
-== Documentation is the single source of truth (SSOT)
+The {ldocs} is the SSOT for all information related to LinuxCNC implementation,
+usage, and troubleshooting.
 
-The GitLab documentation is the SSOT for all
-information related to GitLab implementation, usage, and troubleshooting. It evolves
-continuously, in keeping with new products and features, and with improvements
+It evolves continuously, in keeping with new features, and with improvements
 for clarity, accuracy, and completeness.
 
 This policy prevents information silos, making it easier to find information
-about GitLab products.
+about LinuxCNC applications.
 
 It also informs decisions about the kinds of content we include in our
 documentation.
 
-=== The documentation includes all information
+=== Include all information
 
 Include problem-solving actions that may address rare cases or be considered
-risky, but provide proper context through fully detailed
-warnings and caveats. This kind of content should be included as it could be
-helpful to others and, when properly explained, its benefits outweigh the risks.
-If you think you have found an exception to this rule, contact the
+risky, but provide proper context through fully detailed warnings and
+caveats.
+
+This kind of content should be included as it could be helpful to others
+and, when properly explained, its benefits outweigh the risks. If you
+think you have found an exception to this rule, contact the
 Technical Writing team.
 
-GitLab adds all troubleshooting information to the documentation, no matter how
-unlikely a user is to encounter a situation.
+LinuxCNC adds all troubleshooting information to the documentation, no
+matter how unlikely a user is to encounter a situation.
 
-GitLab Support maintains their own
+////
+FIXME Add Troubleshooting documentation
+LinuxCNC Support maintains their own
 link:../../../administration/index.md#support-team-documentation[troubleshooting content]
-in the GitLab documentation.
+in the LinuxCNC documentation.
+////
 
-=== The documentation includes all media types
+[[docs:style:media]]
+=== Include all media types
 
-Include any media types/sources if the content is relevant to readers. You can
-freely include or link presentations, diagrams, and videos. No matter who
-it was originally composed for, if it is helpful to any of our audiences, we can
-include it.
+Include any media types/sources if the content is relevant to readers.
 
-* If you use an image that has a separate source file (for example, a vector or
-diagram format), link the image to the source file so that anyone can update or reuse it.
+You can freely include or link presentations, diagrams, and videos. No
```

Comment:
Whilst this is a nice idea, we generate PDF documentation, so need to be clear what happens in that case to, for example, an embedded movie.